### PR TITLE
NXP-27489: tracking export start/end time

### PR DESCRIPTION
**Pull request by andreinechaev**

### CODE REVIEW

File: nuxeo-ai-model/src/main/java/org/nuxeo/ai/cloud/AICorpus.java [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+        protected long evaluationDocCount;
+
+        @JsonProperty("query")
+        protected String query;
+
+        @JsonProperty("split")
+        protected int split;
+
+        @JsonProperty("fields")
+        protected List<Map<String, Object>> fields;
```
Comment (PedroCardoso) [inline nuxeo-ai-model/src/main/java/org/nuxeo/ai/cloud/AICorpus.java]: Why `Object` as a type in `Map<String, Object>`
The filed should be a field name and type, being type a enum or a static String.

File: nuxeo-ai-model/src/main/java/org/nuxeo/ai/cloud/AICorpus.java [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+ */
+package org.nuxeo.ai.cloud;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class AICorpus {
```
Comment (covolution) [inline nuxeo-ai-model/src/main/java/org/nuxeo/ai/cloud/AICorpus.java]: You just renamed `AICorpus` to `DatasetExport`, can we give this a better name and some javadoc?

Comment (covolution) [reply nuxeo-ai-model/src/main/java/org/nuxeo/ai/cloud/AICorpus.java]: e.g. POJO for a DocumentType in the cloud

Comment (andreinechaev) [reply nuxeo-ai-model/src/main/java/org/nuxeo/ai/cloud/AICorpus.java]: done

Comment (andreinechaev) [reply nuxeo-ai-model/src/main/java/org/nuxeo/ai/cloud/AICorpus.java]: it's AI_Corpus, I'm not sure what can be a better name :)

**Pull request merged**